qutip/measurement.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of QuTiP: Quantum Toolbox in Python.
+#
+#    Copyright (c) 2011 and later, Paul D. Nation and Robert J. Johansson.
```

Comment:
I think we are putting the (c) to the "QuTiP Project" these days.

Reply:
Corrected.